app/controllers/bulk_ingest_controller.rb

```diff
@@ -163,7 +163,7 @@ def new_pending_uploads
 
       @new_pending_uploads = []
       selected_files.each do |selected_file|
-        file_attributes = selected_file.to_h
+        file_attributes = selected_file.to_h.symbolize_keys
```

Comment:
Can you update the test to pass strings?

Reply:
I am sorry for that, https://github.com/pulibrary/figgy/blob/master/spec/controllers/bulk_ingest_controller_spec.rb#L201 passes the keys as strings, and I now check to ensure that these are symbolized with https://github.com/pulibrary/figgy/pull/3484/files#diff-1248e98671a3d761ac436ef1b0447f78R239